### PR TITLE
[backport v1.0.3] image updates: harvester-upgrade and pre-delete job image

### PR DESCRIPTION
**Pull request by bk201**
backport of https://github.com/harvester/harvester/pull/2579

### CODE REVIEW

**Review by FrankYang0529 — APPROVED** [review]
LGTM

File: deploy/charts/harvester/values.yaml [hunk 1]
```diff
@@ -322,11 +322,11 @@ jobs:
 
           ## Specify the repository of image.
           ##
-          repository: bitnami/kubectl
+          repository: rancher/harvester-upgrade
```
Comment (guangbochen) [inline deploy/charts/harvester/values.yaml]: maybe we can just update the job template to use the `upgrade.image.repository` value directly.

Comment (bk201) [reply deploy/charts/harvester/values.yaml]: Fixed by updating template. Thanks!

**Review by guangbochen — APPROVED** [review]
LGTM, thanks.

**Pull request merged**